packages/terra-application-layout/tests/jest/UserData.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import UserData from '../../lib/user/_UserData';
```

Comment:
Typically we [test src code](https://github.com/cerner/terra-core/blob/master/packages/terra-modal/tests/jest/ModalExample.jsx#L2) in jest. Should we follow that pattern?